### DISCUSSION TITLE
About "Discussions"

### DISCUSSION THREAD

**Discussion by rsc** [golang/go#47139] [Discussions]
The Go project has struggled to find a place to scale discussions that many people may want to participate in. 
See the longer discussion of this problem at https://research.swtch.com/proposals-discuss.

In the past we have used the Go Reddit, mainly because of its threading support. Unfortunately, Reddit (the overall site) is somewhere that many community members feel unwelcome or unsafe.

We are trying out GitHub Discussions as an alternate place to have these kinds of structured discussions, especially for feedback on pre-proposal ideas. It doesn't have arbitrary tree-structured threading, but it does have one-level matching of comments and replies, which might be enough.

For now, we are only adding discussions for very limited use cases, and general users do not have permission to create new discussions. If this works well, we may expand beyond proposal issues that outgrow the linear issue-tracker comment model.

Stay tuned.

**Comment by cespare** [comment]
> For now, we are only adding discussions for very limited use cases, and general users do not have permission to create new discussions.

Last month [GitHub added polls](https://github.blog/2022-04-12-whats-new-in-github-discussions-organization-discussions-polls-and-more/) and it looks like general users *do* have permission do create them. (I noticed #52954.)

Is that something we want to change? (Can it be changed?)
